src/transformer_tsp.py

```diff
@@ -0,0 +1,55 @@
+"""
+transformer_tsp
+"""
+import torch
+from tst import Transformer
+
+from .time_series_predictor import TimeSeriesPredictor
+
+# pylint: disable=too-many-instance-attributes
+class TransformerTimeSeriesPredictor(TimeSeriesPredictor):
```

Comment:
As said before, I'd like to avoid transformer being mentioned in this repo

Reply:
Got it!